ores/logging/logstash_handler.py

```diff
@@ -0,0 +1,21 @@
+
+from logging.handlers import DatagramHandler, SocketHandler
+from .logstash_fomatter import LogstashFormatter
+
+
+class LogstashHandler(DatagramHandler, SocketHandler):
+    """
+    Python logging handler for Logstash.
+
+    :param host: The host of the logstash server.
+    :param port: The port of the logstash server (default 5959).
+    :param message_type: The type of the message (default logstash).
+    :param tags: list of tags for a logger (default is None).
+    """
+
+    def __init__(self, host, port=5959, message_type='logstash', tags=None):
```

Comment:
I don't think there is a default port for the udp protocol. GELF on the other hand which is what we use defaults to 12201 (https://www.elastic.co/guide/en/logstash/current/plugins-inputs-gelf.html) and we actually have it set to 12201.

Reply:
Changed the default, thanks!